src/lib/consts.js

```diff
@@ -6,18 +6,18 @@ const { KEY_VALUE_STORE_KEYS } = require('apify-shared/consts');
 // the name and description should be taken from package.json files in those dirs so that we edit it in a single location!
 // BTW the templates should be clustered by languages, now we have only JS but we'll add other languages later.
 exports.ACTS_TEMPLATES = {
-    basic: {
-        name: 'Basic ("basic")',
-        value: 'basic',
-        dir: `${__dirname}/../templates/basic`,
+    puppeteer_crawler: {
+        name: 'Puppeteer crawler - Boilerplate for recursive crawling with headless Chrome and Puppeteer',
+        value: 'puppeteer_crawler',
+        dir: `${__dirname}/../templates/puppeteer_crawler`,
         defaultRunOptions: {
             build: 'latest',
-            timeoutSecs: 3600,
-            memoryMbytes: 512,
+            timeoutSecs: 0,
+            memoryMbytes: 2048,
         },
     },
     puppeteer: {
-        name: 'Puppeteer ("puppeteer")',
+        name: 'Puppeteer - Boilerplate for loading a web page using headless Chrome and Puppeteer',
```

Comment:
Maybe - `Boilerplate for interaction with a web page using headless Chrome and Puppeteer`